src/use_cases_execution/schedules.rb

```diff
@@ -75,5 +75,11 @@ def self.load
       { path: "#{__dir__}/missing_work_logs/notify_missing_work_logs.rb", interval: 300_000 },
       { path: "#{__dir__}/missing_work_logs/garbage_collector.rb", time: ['14:00'] }
     ].freeze
+
+    OSS_SCORE_SCHEDULES = [
```

Comment:
We need to ensure here this gets executed the last Friday of each month

Reply:
@kren-prog After reviewing with Felipe, we understand an improvement at the orchestator specification needs to be done to support the kind of behavior "im suggestion, so for now this approach works well, sorry for the confusion.